packages/preact-query/eslint.config.js

```
@@ -0,0 +1,26 @@
+// @ts-check
+
+import rootConfig from './root.eslint.config.js'
+// @ts-ignore: no types for eslint-config-preact
+import preact from 'eslint-config-preact'
```

ESLint

[error] `eslint-config-preact` import should occur before import of `./root.eslint.config.js`

(import/order)